src/mpi/coll/op/opequal.c

```diff
@@ -12,11 +12,6 @@ typedef struct MPIR_2int_eqltype {
     int is_equal;
 } MPIR_2int_eqltype;
 
-typedef struct MPIR_floatint_eqltype {
-    float value;
-    int is_equal;
-} MPIR_floatint_eqltype;
-
```

Comment:
Done add changes in one commit and remove them in later commits. Revise the commits so you don't introduce them in the first place.

Reply:
Done